pkg/server/conn_stmt_params_test.go

```diff
@@ -319,6 +319,20 @@ func buildDatetimeParam(year uint16, month uint8, day uint8, hour uint8, minute
 	return result
 }
 
+func buildDatetimeParamWithClientQueryAttr(year uint16, month uint8, day uint8, hour uint8, min uint8, sec uint8, msec uint32) []byte {
```

Comment:
```suggestion
func buildDatetimeParamWithClientQueryAttr(year uint16, month uint8, day uint8, hour uint8, _min uint8, sec uint8, msec uint32) []byte {
```
Maybe `_min` or `_minute` because `min` is a [predeclared identifier](https://go.dev/ref/spec#Predeclared_identifiers)